pallets/manta-pay/src/lib.rs

```diff
@@ -1088,6 +1131,32 @@ pub enum TransferLedgerError {
 
     /// Invalid Transfer Shape
     InvalidTransferShape,
+
+    /// Proof System Error
+    ProofSystemError(ProofSystemError<config::Config>),
+
+    /// Invalid Proof
+    InvalidProof,
+
+    /// Type Marker Parameter
+    Marker(PhantomData<T>),
```

Comment:
This is an extra thing, which no need to do, since we also didn't do it.
But if we can have more descriptive comments for each error it would be great. Currently the name is just repeated, which adds nothing.

Reply:
I added a few explanations for the errors which are not unexpected.